### ISSUE TITLE
"Show Original" icon has very poor contrast on dark background

### ISSUE THREAD

**Issue by holly-hacker** [ajayyy/DeArrow#205]
Currently, the "Show Original" icon looks like this:
| Original title | Replaced title |
| - | - |
| ![unchanged](https://github.com/ajayyy/DeArrow/assets/13605369/d66e8e9f-ae95-4821-adfd-33a9687af365) | ![replaced](https://github.com/ajayyy/DeArrow/assets/13605369/e6b0b6c9-47f7-46af-b341-752de5391627) |

According to #191, users should be able to tell which is which through the amount of rings, but this can be really hard to see on a dark background (especially for those with impaired vision or some form of colorblindness).

I'd like to suggest an option to change the colors of the "replaced" variant of the icon to something that is easier to tell apart from the other icon.

As an example, this is more visible (using color codes `#e9a316`, `#f9d386` and `#a4710a`):
![orange icon](https://github.com/ajayyy/DeArrow/assets/13605369/cb832012-b59c-47ae-bdbf-06c623bd1b2a)

**Comment by khangquangtran** [comment]
Yeah. I find the gray (for original title) and blue (for modified title) pretty hard to distinguish. Maybe gray and red (the theme color of YouTube) can be more "visible".
